### ISSUE TITLE
Mobile UX/UI for GitHub post info

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#2373]
In #2367 we are adding GitHub info to the sidebar for a post.  Currently, it only works in desktop, and there's nothing visible in mobile.  I'd like to see if we can add a design and UI for the mobile case.

One option might be to show a collapsed GitHub area that expands when you touch it.  If there's no GitHub details, we wouldn't show it.  But if a post does have any, we'd include a GitHub icon or something that you can use to drill down and see more.

Obviously we don't want to show this all the time, but it would be good if you could access it on demand.

**Comment by irenejoeunpark** [comment]
Hi @humphd, If no one has started working on this, can I start contributing to the UI implementation for mobiles?

**Comment by humphd** [comment]
@irenejoeunpark that would be great, please do.  I would encourage you to chat with people in the #telescope channel on Slack, as many of the devs/users will have opinions about this, so getting community input early will be key.  Also, we can do this in stages (e.g., you don't have to solve the whole thing in one PR).

Looking forward to seeing this happen!

**Comment by sirinoks** [comment]
We have just completed changing a column view of isses/PRs to a row and fixed its wrapping. Await for the PR for that one. @TueeNguyen should make the PR.

There is another task to impove the view, which turned out to be more difficult than we first thought. I will create another issue describing it.
